sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -55,12 +55,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.14.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.15.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
We should use the [unreleased_](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L222) instead as they have additional guards preventing libraries from being shipped with them. Basically, it catches scenarios where `current` dependencies would allow shipping but that dependency version hasn't been released.